sbin/adduser_course.py

```diff
@@ -107,6 +111,46 @@ def main():
             registration_section=registration_section
         )
 
+    # function taken from setup_sample_courses
```

Comment:
Won't the database do all of this for us now?

Reply:
Thanks for pointing that, missed to revert these changes after adding the trigger.